From 1d18148c86377787a8c77042b12ea66f20cb2ca9 Mon Sep 17 00:00:00 2001
From: Tobias Markmann <tm@ayena.de>
Date: Thu, 21 Jun 2018 13:04:56 +0200
Subject: Add missing include for QAbstractItemModel

This fixes building Swift with Qt 5.11.

Test-Information:

Builds and tests pass on macOS 10.13.5 with Qt 5.11.0.

Change-Id: I1be2cd081d8a520ec38ab7cca5ada0d7fc39b777

diff --git a/Swift/QtUI/UserSearch/QtUserSearchWindow.h b/Swift/QtUI/UserSearch/QtUserSearchWindow.h
index 0714ac1..fe536ab 100644
--- a/Swift/QtUI/UserSearch/QtUserSearchWindow.h
+++ b/Swift/QtUI/UserSearch/QtUserSearchWindow.h
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2010-2016 Isode Limited.
+ * Copyright (c) 2010-2018 Isode Limited.
  * All rights reserved.
  * See the COPYING file for more information.
  */
@@ -8,6 +8,7 @@
 
 #include <set>
 
+#include <QAbstractItemModel>
 #include <QWizard>
 
 #include <Swiften/Base/Override.h>
-- 
cgit v0.10.2-6-g49f6

